get_coverage_for_challenge.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -u
+set -o pipefail
+
+SCRIPT_CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+CHALLENGE_ID=$1
+NODEJS_TEST_REPORT_JSON_FILE="${SCRIPT_CURRENT_DIR}/coverage/coverage-summary.json"
+NODEJS_CODE_COVERAGE_INFO="${SCRIPT_CURRENT_DIR}/coverage.tdl"
+
+( cd ${SCRIPT_CURRENT_DIR} && npm install && npm run coverage || true 1>&2 )
+
+[ -e ${NODEJS_CODE_COVERAGE_INFO} ] && rm ${NODEJS_CODE_COVERAGE_INFO}
+
+if [ -f "${NODEJS_TEST_REPORT_JSON_FILE}" ]; then
+    node_modules/prettier/bin-prettier.js --write ${NODEJS_TEST_REPORT_JSON_FILE}
```

Comment:
User the ${SCRIPT_CURRENT_DIR} when calling using relative path, should not assume the path.

Reply:
Done!